tests/drivers/spi/spi_loopback/testcase.yaml

```diff
@@ -87,9 +91,9 @@ tests:
     platform_allow:
       - nucleo_h743zi
       - nucleo_h753zi
-  drivers.spi.stm32_spi_16bits_frames_dma_no_nocache.loopback:
+  drivers.spi.stm32_spi_16bits_frames_dma_dt_nocache_mem.loopback:
     extra_args:
-      - OVERLAY_CONFIG="overlay-stm32-spi-16bits-dma-no-nocache.conf"
+      - OVERLAY_CONFIG="overlay-stm32-spi-16bits-dma-dt-nocache-mem.conf"
```

Comment:
This files holds `CONFIG_SPI_LOOPBACK_16BITS_FRAMES=y` so adding  - DTC_OVERLAY_FILE="overlay-stm32-spi-16bits.overlay" is superfluous. Some clean up could be used.

Reply:
`overlay-stm32-spi-16bits.overlay` is needed in this test case bcz `dmas`SPI property contains special 16bits flags.